test/authentication/resolver/userauth.resolver.test.ts

```diff
@@ -81,6 +82,13 @@ describe('Userauth Module', () => {
           username: 'user@test.com',
           password: 's3cr3t1234567890',
         };
+        const user = {
+          id: 'ae032b1b-cc3c-4e44-9197-276ca877a7f8',
```

Comment:
Let's not reintroduce a new string. Let's reuse the existing `users[]`. The only difference is, this local variable will not have the password and other unwanted details.

Example: [Code reference](https://github.com/KeyValueSoftwareSystems/authentication-service/blob/ea9b3ec6593f92ae1b545c75aed6f32ebbc5dfed/test/authentication/resolver/userauth.resolver.test.ts#L124-L130)